multimedia/tvheadend/Makefile

```diff
@@ -26,120 +22,215 @@ include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
 define Package/tvheadend
```

Comment:
Here should be two spaces instead of tab.

Reply:
Sorry. I had the impression there must be tabs everywhere. I hope it's fixed now.
I can't find a guide anywhere about tabs vs spaces.